### ISSUE TITLE
[Kernel] please publish kernel version every time after code changes

### ISSUE THREAD

**Issue by lfkdsk** [dart-lang/sdk#41599]
This tracker is for issues related to:
* Common Front End (CFE) and kernel

Dart Community may need to use kernel package to do some optimize or codegen (just like us).  But kernel package hasn't published a new version since v0.3.29 (Oct 31, 2019) : 

![image](https://user-images.githubusercontent.com/11584398/79857523-7ec61a80-8400-11ea-9cf6-2f4a74a64b8e.png)

So could dart dev publish every kernel version (after format version or other changes) to the community?

(depend it on git will clone all dart-lang sdk, it's too large for dev usage)

**Comment by devoncarew** [comment]
I don't believe these packages are generally intended to be consumed by users (publishing them was necessary as part of publishing package:analyzer). The CFE team can confirm.

**Comment by lfkdsk** [comment]
@devoncarew 'package: analyzer' can analyze dart file (analyzer package contains dart ast nodes, not kernel ast nodes), but I just want to use API in `kernel` just like read/write kernel binary.

**Comment by lfkdsk** [comment]
DSL based on DartVM, or other compile-time Codegen is straightly based on `package: kernel`(some packages were published on GitHub ), so it can be consumed by a part of users and necessary to be published.
